src/main/java/com/minecolonies/coremod/colony/jobs/JobLumberjack.java

```diff
@@ -93,7 +97,18 @@ public void onLevelUp()
             AttributeModifierUtils.addModifier(worker, speedModifier, Attributes.MOVEMENT_SPEED);
         }
     }
-    
+
+    @Override
+    public boolean onStackPickUp(final @NotNull ItemStack pickedUpStack)
+    {
+        final boolean result = super.onStackPickUp(pickedUpStack);;
+        if (getCitizen().getRandom().nextInt(Constants.STACKSIZE) <= 1)
```

Comment:
please don't abuse existing constants for a different purpose, if we were to adjust stacksize we'd accidentally change the misteltoe chance xD

Reply:
It could, though.  In fact it probably shouldn't even exist, the things that use it should be using `Item.getMaxStackSize()` instead.

Reply:
Yeah, but in some instances we want to do calc with the usual 64, and don't have a particular item.

Reply:
Sure, there's a couple places that use a tag or a predicate, but the vast majority of uses are in the context of a single item...
not that this is directly relevant to this particular feedback.  I agree with someaddons, though, even if STACKSIZE never changes, a coincidence of value is a poor justification for using a constant in an entirely unrelated context.

Reply:
But it's not a random constant. I want it to drop once every absolute max stack size